src/cli/measure_manager.rb

```diff
@@ -468,24 +470,42 @@ def get_arguments_from_measure_info(measure_info)
       elsif type == "Double".to_OSArgumentType
         arg[:units] = argument.units.get if argument.units.is_initialized
         arg[:default_value] = argument.defaultValueAsDouble if argument.hasDefaultValue
-
+        if argument.hasDomain
```

Comment:
Does it still have domain if either min or max are set but not both?

Reply:
Domain is set as `[std::numeric_limits<double>::lowest(), max]` if max only is provided, and if min only then `[min, std::numeric_limits<double>::max()]`